CONTRIBUTING.md

```diff
@@ -19,7 +19,6 @@ ask you to make any refinements needed or merge it and make them ourselves.
 
 Things that should go into your PR description:
 
--   A changelog entry in the `Notes` section (see below)
```

Comment:
This felt misleading as most of the time it's not necessary. Instead, I embellished a little in the relevant section.